parsers/myrddin.c

```diff
@@ -0,0 +1,53 @@
+/*
+ *   Copyright (c) 2016, Ori Bernstein
+ *
+ *   This source code is released for free distribution under the terms of the
+ *   GNU General Public License version 2 or (at your option) any later version.
+ *
+ *   This module contains functions for generating tags for Myrddin language
+ *   files. (https://myrlang.org)
+ */
+
+/*
+ *   INCLUDE FILES
+ */
+#include "general.h"  /* must always come first */
+#include "parse.h"
+
+static const tagRegexTable myrddinTagRegexTable [] = {
+	/* const foo = {; function literal } */
+	{"^([ \t]*extern)?[ \t]*const[ \t]+([a-zA-Z_][a-zA-Z0-9_]*)[ \t]*=[ \t]{",
```

Comment:
(Look at the result of test running on Appveyor and Travis-Ci.
You can run the test with "make units LANGUAGES="Myrddin")

A backslash may be needed before `{` because `{` is a meta character.